### ISSUE TITLE
Exception of type 'System.Exception' was thrown

### ISSUE THREAD

**Issue by Strahbehry** [firegiant/HeatWaveSupport#61]
### HeatWave Version

1.0.4

### Visual Studio Version

17.9.6

### Repro Steps

1. Open Visual Studio
2. Attempt to make a MSI installer package

### Actual Result

![image](https://github.com/firegiant/HeatWaveSupport/assets/15874111/06a97edc-9022-48d0-a948-311bc765a59d)

**Comment by Strahbehry** [comment]
[vsconfig.txt](https://github.com/firegiant/HeatWaveSupport/files/14975559/vsconfig.txt)
[MyVSLog.txt](https://github.com/firegiant/HeatWaveSupport/files/14975562/MyVSLog.txt)

had to upload both as .txt but one is .vsconfig and the other ActivityLog.xml

**Comment by peterwilson69** [comment]
I'm getting the exact same error. Did this ever get resolved?

**Comment by Strahbehry** [comment]
Nope, there was this issue before but it got closed because it was presumed fixed. But no acknowledgment yet of the persisting issue https://github.com/firegiant/HeatWaveSupport/issues/59

**Comment by cjt-westwoodps** [comment]
I have the same error originally reported on issue #48.

The problem persists with VS 17.10 and HeatWave version 1.0.4.

### HeatWave Version
1.0.4

### Visual Studio Version
17.10.0

### Repro Steps
1. Open Visual Studio
2. Attempt to make a MSI installer package

Actual Result
![image](https://github.com/firegiant/HeatWaveSupport/assets/125506967/307a16a8-23bc-44cd-8229-ef96f57121be)

[vsconfig.txt](https://github.com/firegiant/HeatWaveSupport/files/15476397/vsconfig.txt)

**Comment by MatthiasRInst** [comment]
I cannot create Bundle, MsiPackage, WixLib, or Merge Module projects with HeatWave, getting that System.Exception in all the 4 cases (Custom Action Projects work.)

I am using:

Visual Studio 17.9.7
HeatWave 1.0.4.5

I have attached both my .vsconfig and an AcitivityLog.xml from a session in which I tried to create 4 Projects one after the other:
Bundle
MsiPackage
WiXLib
MergeModule.

The errors start with record 836.

[20240626VsConfig.txt](https://github.com/user-attachments/files/16012838/20240626VsConfig.txt)
[20240627VSActivityLog.txt](https://github.com/user-attachments/files/16012846/20240627VSActivityLog.txt)

**Comment by KIWI1982** [comment]
Bonjour,

J'aimerai savoir si ce problème a été réellement résolu sur Visual Studio 2022 et Heatware 1.0.4

J'aimerai utiliser votre outil mais je suis finalement bloqué car je ne peux pas faire un projet :)

**Comment by Julie-BC** [comment]
I think this is the same issue, same fix : https://github.com/firegiant/HeatWaveSupport/issues/69#issuecomment-2386263768

**Comment by FireGiantHelp** [comment]
We were never able to reproduce this issue. However, one of our customers did encounter the problem when using Visual Studio 17.10. During the investigation, the customer upgraded to Visual Studio 17.12.5 and the issue disappeared. Our hypothesis is that Visual Studio addressed a failure when resolving Sdk-style projects when disconnected to the internet with an error message in 17.12.

HeatWave v1.0.5 is out with support for 17.12 and 17.13. Try upgrading to the latest Visual Studio LTS or current and we believe this particular error is resolved.
